src/single_instruction_test/program.rs

```diff
@@ -36,6 +36,10 @@ impl<'a> Arbitrary<'a> for Program {
 }
 
 impl Program {
+    pub fn new(_instructions: Vec<Instruction>, _code_page: Vec<U256>) -> Self {
+        unimplemented!("Not used during fuzzing")
```

Comment:
Then why add the method?

Reply:
Needed for compilation of integration tests in the main library to succeed. There's a way to signal that a test doesn't work with a particular feature (add `#![cfg(not(feature = ".."))]` at the start of the file), but since there are other dummy methods added to make the library itself work, I've decided that this would be a preferred approach. WDYT? I don't think that having integration tests not compiling at all is a reasonable option, at least when we decide to publish the library.